cluster/local/microk8s.sh

```diff
@@ -52,6 +52,10 @@ case "${1:-}" in
 
     microk8s.enable registry
 
+    # We'll use locally cached image instead of having it downloaded by kind
+    # cluster.
+    docker pull "gcr.io/kubernetes-helm/tiller:${HELM_VERSION}"
+    copy_image_to_cluster "gcr.io/kubernetes-helm/tiller:${HELM_VERSION}" "gcr.io/kubernetes-helm/tiller:${HELM_VERSION}"
```

Comment:
I believe you are going to want to make the second image name here correspond to the microk8s registry. I _think_ this will attempt to just push it back to gcr https://microk8s.io/docs/registry-built-in

Reply:
Thanks for catching! I've changed the copy_image_to_cluster function to prepend `localhost:32000` as documented in the link you provided.

Reply:
Me neither and I really suspect that even anyone is using this.